### PR TITLE
charls

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/charls/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: charls
+  version: {{ version }}
+
+source:
+  url: https://github.com/team-charls/charls/archive/{{ version }}.tar.gz
+  sha256: 528c6a3cc168a44e73f2890d8f4a35104a54d752eba3d6a643f050b72dd67cfa
+  fn: charls-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/charls/meta.yaml]:
```suggestion
```

File: recipes/charls/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make   # [not win]
+    - ninja  # [win]
+  host:
+    # Dummy requirement
```
Comment (chrisburr) [inline recipes/charls/meta.yaml]: Why is a dummy requirement needed?

Comment (hmaarrfk) [reply recipes/charls/meta.yaml]: something was complaining.

**Comment by hmaarrfk** [comment]
thx

**Pull request merged**